compiler-rt/lib/sanitizer_common/sanitizer_common_interceptors.inc

```diff
@@ -3917,7 +3938,11 @@ INTERCEPTOR(SIZE_T, wcrtomb, char *dest, wchar_t src, void *ps) {
   if (res != ((SIZE_T)-1)) {
     CHECK_LE(res, sizeof(local_dest));
     COMMON_INTERCEPTOR_WRITE_RANGE(ctx, dest, res);
+#  if !SANITIZER_AIX
```

Comment:
local_dest is small
switching always to internal_memcpy should be fine

also selarate PR would be nice